lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/telemetry/TelemetryAppender.java

```diff
@@ -0,0 +1,83 @@
+package org.enso.logging.service.logback.telemetry;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.AppenderBase;
+import com.fasterxml.jackson.core.exc.StreamReadException;
+import com.fasterxml.jackson.databind.DatabindException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+// TODO: Presunout sem LogJob, apod.
+
+public final class TelemetryAppender extends AppenderBase<ILoggingEvent> {
+  private static final String CREDENTIALS_FILE_ENV = "ENSO_CLOUD_CREDENTIALS_FILE";
```

Comment:
```suggestion
  private static final String CREDENTIALS_FILE_ENV = "ENSO_CLOUD_CREDENTIALS_PATH";
```

Reply:
The name of the env var is consistent with authentication in [Standard.Base.Enso_Cloud](https://github.com/enso-org/enso/blob/244effde0ca274983c12d9324947dce299be1fce/distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Internal/Authentication.enso#L59)